pkg/logbridge/logbridge.go

```diff
@@ -0,0 +1,88 @@
+package logbridge
+
+import (
+	"bufio"
+	"io"
+
+	"github.com/square/p2/pkg/logging"
+)
+
+// This type is agnostic of pipes. os.File conforms to io.Reader and io.Writer
+// so it should not be difficult to migrate
+type Bridge struct {
+	src  io.Reader
+	dest io.Writer
+
+	lines         chan []byte
+	needsTeardown bool
+}
+
+// New takes two files src and dest and returns a composite object of the
+// two.
+func New(src io.Reader, dest io.Writer, capacity int) Bridge {
```

Comment:
The functionality of the logbridge is very similar to [`io.Copy()`](https://golang.org/pkg/io/#Copy), but that function takes its arguments in the other order:  `func Copy(dst Writer, src Reader) (written int64, err error)`.  Can you change the order of your arguments to match the standard library?